lib/jsonapi/utils/response/formatters.rb

```diff
@@ -186,6 +186,8 @@ def turn_into_resource(record, options)
         # @api private
         def result_options(records, options)
           {}.tap do |data|
+	          data[:meta] = options.fetch(:meta, {})
```

Comment:
Not sure if `data[:meta]` has record counts already, but id maybe merge the two instead of assigning

Reply:
```suggestion
	          data[:meta] = data.fetch(:meta, {}).merge(options.fetch(:meta, {}))
```